pkgs/applications/emulators/wine/sources.nix

```diff
@@ -42,6 +50,25 @@ in rec {
       # Also look for root certificates at $NIX_SSL_CERT_FILE
       ./cert-path.patch
     ];
+
+    updateScript = writeShellScript "update-wine-stable" (''
+      ${updateScriptPreamble}
+      major=''${UPDATE_NIX_OLD_VERSION%.*}
+      latest_stable=$(get_latest_wine_version "$major.0")
+      latest_gecko=$(get_latest_lib_version wine-gecko)
+    '' +
+      # Can't use autobump on stable because we don't want the path
+      # <source/7.0/wine-7.0.tar.xz> to become <source/7.0.1/wine-7.0.1.tar.xz>.
+    ''
```

Comment:
```suggestion

    # Can't use autobump on stable because we don't want the path
    # <source/7.0/wine-7.0.tar.xz> to become <source/7.0.1/wine-7.0.1.tar.xz>.
```